spec/rubocop/cop/sorbet/single_line_rbi_class_module_definitions_spec.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe(RuboCop::Cop::Sorbet::SingleLineRbiClassModuleDefinitions, :config) do
+  subject(:cop) { described_class.new(config) }
+
+  describe('offences') do
+    it 'registers an offense when an empty module definition is split across multiple lines' do
+      expect_offense(<<~RBI)
+        module MyModule
+        ^^^^^^^^^^^^^^^ Empty class/module definitions in RBI files should be on a single line.
+        end
+
+        module SecondModule
+        ^^^^^^^^^^^^^^^^^^^ Empty class/module definitions in RBI files should be on a single line.
+        end
+      RBI
+    end
+
+    it 'registers an offence when an empty class definition is split across multiple lines' do
+      expect_offense(<<~RBI)
+        class MyClass
+        ^^^^^^^^^^^^^ Empty class/module definitions in RBI files should be on a single line.
+        end
+
+        class AnotherClass < SomeParentClass
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Empty class/module definitions in RBI files should be on a single line.
+        end
+      RBI
+    end
+  end
+
+  describe('no offences') do
+    it 'does not register an offense when empty module definition is done on a single line' do
+      expect_no_offenses(<<~RBI)
+        module MyModule; end
+
+        module AnotherModule; end
+      RBI
+    end
+
+    it 'does not register an offense when empty class definition is done on a single line' do
+      expect_no_offenses(<<~RBI)
+        class MyClass; end
+
+        class AnotherClass < SomeParentClass; end
+      RBI
+    end
+
+    it 'does not register an offence when a module is not empty' do
+      expect_no_offenses(<<~RBI)
+        module MyModule
+          def hello; end
+        end
+      RBI
```

Comment:
Can we also test with a multi-line method body:
```suggestion
        module MyModule
          def hello; end
        end

        module AnotherModule
          def world
          end
        end
      RBI
```

Reply:
Good call!